tests/chainer_tests/functions_tests/loss_tests/test_softmax_cross_entropy.py

```diff
@@ -13,24 +13,43 @@
 from chainer.testing import condition
 
 
+@testing.parameterize(*testing.product({
```

Comment:
It is too large. I requires about two minutes. How about testing random combination?